coverage_test.go

```diff
@@ -95,16 +94,12 @@ func checkMatch(t *testing.T, m Match, name string, prevEnd int) {
 		return
 	}
 	lic := findLicense(name)
-	// Skip leading white space, almost certainly trimmed copyright lines.
 	length := len(lic.doc.text)
-	for _, c := range lic.doc.text {
-		if c != ' ' && c != '\n' {
-			break
-		}
-		length--
-	}
 	// There is some fudge factor in the match lengths because of terminal spaces, so be forgiving.
 	min, max := length-5, length
+	if name == "BSD-2-Clause" {
```

Comment:
why? comment. this looks like a hack.

Reply:
It was needed before I added the proper wildcard support. Removed.